packages/storybook8/stories/Concepts/AdhocCalling/Docs.mdx

```diff
@@ -0,0 +1,65 @@
+import { MessageBar, MessageBarType } from '@fluentui/react';
+import { Meta, Source } from '@storybook/addon-docs';
+
+import AdhocCallingSnippetText from '!!raw-loader!./snippets/AdhocCalling.snippet.tsx';
+import AdhocCallingVoiceAppSnippetText from '!!raw-loader!./snippets/AdhocCallingVoiceApp.snippet.tsx';
+
+<Meta title="Concepts/Ad-hoc calling" />
+
+# Ad-hoc calling
+
+## Overview
+
+A call can be started with a Teams user or a call queue by using their id. You can call a teams user or a Teams voice application
+like a call queue or auto attendant. These calls it is important to note can only be one Teams voice application, or multiple teams users. You cannot call multiple
+Teams voice applications at the same time.
+
+## Calling a Teams voice application
+
+To call a Teams voice application like a call queue or auto attendant, you need to use the id of the Teams resource account. Check out this [tutorial](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/get-started-teams-auto-attendant#find-object-id-for-auto-attendant) on how to find
+this id. The id of the Teams resource account should be stored in the type of `MicrosoftTeamsAppIdentifier[]` and given to the `targetCallees` property
+of the [CallAdapter](?path=/docs/composite-adapters--page#call-adapter) when creating a call. Now when the [CallComposite](?path=/docs/composites-call-basicexample--basic-example) starts up it will call the Teams voice application
```

Comment:
```suggestion
of the [CallAdapter](?path=/docs/composites-adapters--docs#call-adapter) when creating a call. Now when the [CallComposite](?path=/docs/composites-callcomposite--docs) starts up it will call the Teams voice application
```